components/simplified-grid/simplified-grid.css

```diff
@@ -0,0 +1,28 @@
+@import url("/docs/main.css");
+
+.grid_container {
+    display: grid;
+    background-color: var(--light-grey-color);
+    width: fit-content;
+    height: fit-content;
+}
+
+.grid_item {
+    padding: 0.5em;
+} 
+
+.grid_img {
+    height: 100%;
+}
+
+.grid_two_columns {
+    grid-template-columns: 1fr 1fr;
+    gap: 10px;
+    padding: 1rem;
+}
+
+.grid_three_columns {
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: 10px;
+    padding: 0.5rem;
```

Comment:
you can use gap and padding in .grid itself as they are common for both grids.
Rest all is good.

Reply:
Basically, based on need I can just change accordingly.